addons/crm/models/crm_lead.py

```diff
@@ -1173,29 +1173,102 @@ def _get_rainbowman_message(self):
             return _('Phew, that took some effort — but you nailed it. Good job!')
 
         query = """
-            SELECT
```

Comment:
@reth-odoo Was this your suggestion? Have to say I prefer the shorter format of the previous version, but maybe performances are a lot better here? Unsure ^^